rcssserver3d/data/rsg/agent/ball.rsg

```diff
@@ -2,12 +2,13 @@
 
 (RSG 0 1)
 (
+ (templ $X $Y)
```

Comment:
what is temp1?

Reply:
is this how parameters get names???

Reply:
oh it's templ (L not 1).  how did you figure that out?  i see no documentation for this

Reply:
It's used by another 45 .rsg files in that directory... I can't find any documentation either